shellcheck/Dockerfile

```diff
@@ -4,7 +4,8 @@ RUN dnf -y --nodocs  --setopt=install_weak_deps=False \
 --disablerepo=fedora-cisco-openh264 \
 --disablerepo=fedora-modular \
 --disablerepo=updates-modular \
-install ShellCheck findutils file
+install ShellCheck findutils file; \
```

Comment:
shouldn't be there `&&` instead `;` ?

Reply:
everywhere in this PR

Reply:
🤔 
Not sure... local tests worked, but I'll check it again

Reply:
Fixed

Reply:
well, I mean, is there reason to continue when previous command fails?

```
$ test 0 -eq 1 ; echo "yes"
yes

$ test 0 -eq 1 && echo "yes"  
```